### PR TITLE
Node sass support

### PR DESCRIPTION
**Pull request by justame**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -176,7 +180,12 @@ module.exports = function (grunt, options) {
     'autoprefixerIfEnabled',
     'styleInlineServeIfEnabled',
     'newer:copy:vm'
-  ]);
+  ];
+
+  if(options.useNodeSass){
```
Comment (mastertheblaster) [inline Gruntfile.js]: @justame There is another approach for making steps conditional.

```javascript
grunt.registerTask('pre-build', [..., 'wix-sass']);
grunt.registerTask('wix-sass', function () {
  if (grunt.config.useNodeSass) {
    grunt.runTask('sass:dist');
  }
});
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wix-gruntfile",
   "description": "Wix Gruntfile.",
-  "version": "0.1.323",
+  "version": "0.1.324",
```
Comment (mastertheblaster) [inline package.json]: @justame  I guess you need to rebase your pull request.

**Comment by mastertheblaster** [comment]
@justame I guess you need to rebase and why have included npm-shrinkwrap.json ? It locks the dependencies. IMHO not good.

**Comment by shahata** [comment]
plz rebase. I suggest removing shrinkwrap and version change in package.json before rebasing to avoid git conflicts

**Comment by shahata** [comment]
I also suggest squashing before rebase

File: grunt-sections/transform-css.js [hunk 3]
```diff
@@ -47,6 +49,29 @@ module.exports = function (grunt, options) {
...
+    },
+    sass: {
+      dist: {
+        cwd: 'app',
+        src: [(options.useModulesStructure ? 'modules' : 'styles') + '/**/*.scss', '!' + (options.useModulesStructure ? 'modules' : 'styles') + '/**/*.compass.scss'],
+        dest: '.tmp',
+        expand: true,
+        ext: '.css',
+        options: {
+          outputStyle: 'compact',
```
Comment (argshook) [inline grunt-sections/transform-css.js]: @justame why not `compressed`?

**Pull request closed without merging**